### DISCUSSION TITLE
Support font family

### DISCUSSION THREAD

**Discussion by robyhuzwandar** [diegomura/react-pdf#2132] [Q&A]
Hi, is that mean when I use the font in the list below I don't need to register?

<img width="905" alt="image" src="https://user-images.githubusercontent.com/20414047/210137451-25a413a7-7e6a-4d6b-b2a3-fca92cae782e.png">

**Comment by CoskunCakir** [comment]
hi @robyhuzwandar, correct, you don't need to register those fonts.
